docs-2.0/3.ngql-guide/7.general-query-statements/6.show/9.show-parts.md

```diff
@@ -0,0 +1,53 @@
+# SHOW PARTS
+
+The `SHOW PARTS` statement shows the information of a specified partition or all partitions in a graph space.
+
+## Syntax
+
+```ngql
+SHOW PARTS [<part_id>]
+```
+
+## Examples
```

Comment:
put "SHOW PARTS" before "SHOW PARTS 1"? As a user, when I read "SHOW PARTS 1" example, I would want to know how to get the partition number.